site/_posts/2021-04-01-why-user-multi-factor-authentication.md

```diff
@@ -0,0 +1,129 @@
+
+From Section CC6.1 of the SOC document: "Persons, infrastructure, and software are identified and authenticated prior to accessing information assets, whether locally or remotely". The document omits specific implementation details. Talk to your SOC2 auditor about required controls.
+
+### When actions look suspicious
+
+An auth system has a lot of information about who is logging in. During authentication, credentials are supplied; the end result is the answer to this question: "Is the user behind this authentication request who they are claiming to be?" 
+
+Some data, such as the username and password, is explicitly provided to the system. But there is also interesting implicit data available such as:
+
+* The date and time of access
+* Connection information such as IP address and user agent
+* How many times the user has logged in recently
+* Has this device been used to access this system before
+
+Such implicit data can help determine if the authentication request is legitimate. For instance, if a user accesses a system from the USA and 24 hours later there is another request from Germany with the same credentials, is the latter valid? Could be, but it could also be that the request from Germany is due to a compromised account. Requiring MFA at such a time allows you to apply extra scrutiny to strange requests.
+
+Another example where you might want additional authentication factors is when a user logs in from a new device. Popular SaaS applications such as Google's GSuite and Mailchimp can be configured to require MFA for only the initial authentication from a new device.
+
+### Applying MFA requirements
+
+Put the MFA requirement configuration in a central location in your application to make it easier to modify. Tying it tightly to the authentication system generally makes sense. Choices about when to require MFA change over time as a system evolves. New features are built. Different types of users are supported.
+
+This policy configuration should itself be protected and only allow access to privileged users with recent multi-factor authentication.
+
+If you don't have organization or user wide policies, at the least allow users to set up MFA. Doing so allows security conscious users to protect their account. They can evaluate their own security posture and make their own sensible choices. For example, one person may run their entire life through their Gmail account, while others may use Gmail only occasionally as a throwaway email address. In the former case, MFA makes sense; in the latter it may not.
+
+If you are worried about what a compromised account could do to either your systems or your users' data, you should strongly encourage or require MFA.
```

Comment:
Do we want a segue into FusionAuth MFA or is that beyond the scope of this post?  

If you are worried about what a compromised account could do to either your systems or your users' data, you should strongly encourage or require MFA.  Good thing FusionAuth has built this for you.  Read more about it *here*

☝🏻 or something like that...

Reply:
Well, we don't yet have a page for the new MFA goodness that is coming soon. I'll leave this open so if we have a release announcement we can slot a link in here.

But in general I struggle with how hard to push FusionAuth features on the blog posts. I find as a developer I respond better to educational content, but I understand the perspective of "it's a company blog, of course it has a CTA at the end of the post"  too :) .